docs/table-migration.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Overview"
+url: table-migration
+menu:
+  main:
+    parent: "Migration"
+    weight: 100
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+# Table Migration
+Apache Iceberg supports converting existing tables in other formats to Iceberg tables. This section introduces the general concept of table migration, its approaches, and existing implementations in Iceberg.
+
+## Migration Approach
+There are two primary methods for executing table migration: full data migration and in-place migration. In this doc, we will describe more about in-place migration.
```

Comment:
I think we still need to talk a bit about "full data migration", in this case users use operations like CTAS, INSERT, Change data capture pipelines. And you can link to related docs.

Reply:
Thank you for your suggestion. I revised the sentence and added link to CTAS and INSERT. However, it seems currently there is no OSS doc directly describing the Change-Data-Capture pipeline. So I leave it unlinked. I think it shall be ok since CDC pipeline is more like a conception compared to the first two terms which are more like specific implementations.